crml/cennzx-spot/src/impls.rs

```diff
@@ -171,7 +170,11 @@ pub(crate) mod impl_tests {
 					51,
 					&TestFeeExchange::new_v1(TRADE_ASSET_A_ID, 2_000_000),
 				),
-				"Failed to charge transaction fees during conversion"
+				DispatchError::Module {
```

Comment:
I think you can import this error type instead of reconstruct it
```
use crate::Error;
// ..
assert_err!(
   // ..
  Error<Test>::InsufficientTradeAssetReserve
);
```